src/serac/physics/solid.cpp

```diff
@@ -516,27 +504,24 @@ const FiniteElementState& Solid::solveAdjoint(FiniteElementDual& adjoint_load,
   // Set the mesh nodes to the reference configuration
   mesh_.NewNodes(*reference_nodes_);
 
-  adjoint_load.initializeTrueVec();
-
   // note: The assignment operator must be called after the copy constructor because
   // the copy constructor only sets the partitioning, it does not copy the actual vector
   // values
-  mfem::HypreParVector adjoint_load_vector(adjoint_load.trueVec());
-  adjoint_load_vector = adjoint_load.trueVec();
+  mfem::HypreParVector adjoint_load_vector(adjoint_load);
+  adjoint_load_vector = adjoint_load;
```

Comment:
was this redundant initialization pattern addressed by https://github.com/mfem/mfem/pull/2639 ?